pkg/apis/cluster/v1alpha1/cluster.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=kuratorclusters,shortName=kcluster
+// +kubebuilder:subresource:status
+
+// KuratorCluster is the schema for existing node based Kubernetes Cluster API.
+type KuratorCluster struct {
```

Comment:
prefer to user `Cluster` directly

Reply:
The only concern is if we use cluster, it will be same as cluster-api cluster

Reply:
`CustomCluster` is better.